src/App.css

```diff
@@ -21,37 +127,184 @@
 
 .task {
   background: white;
-  border-radius: 3px;
-  box-shadow: 0 1px 3px rgba(0, 0, 0, 0.1);
-  padding: 10px;
+  border-radius: 4px;
+  box-shadow: 0 1px 2px rgba(0, 0, 0, 0.1);
+  padding: 12px 16px;
   margin-bottom: 8px;
   cursor: grab;
+  border: 1px solid var(--border-color);
+  color: var(--text-primary);
+  font-size: 14px;
+  transition: all 0.2s ease;
+}
+
+.task:hover {
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1);
+  border-color: #c0c4c8;
 }
 
 .task:active {
   cursor: grabbing;
+  background: #f8f9fa;
+}
+
+/* Scrollbar customization */
+::-webkit-scrollbar {
+  height: 8px;
+  width: 8px;
+}
+
+::-webkit-scrollbar-track {
+  background: #f1f1f1;
+  border-radius: 4px;
+}
+
+::-webkit-scrollbar-thumb {
+  background: #c1c1c1;
+  border-radius: 4px;
+}
+
+::-webkit-scrollbar-thumb:hover {
+  background: #a8a8a8;
+}
+
+/* Responsive adjustments */
+@media (max-width: 768px) {
+  .app {
+    padding: 16px;
+  }
+  
+  .forms-container {
+    flex-direction: column;
+  }
+  
+  .task-form,
+  .column-form {
+    width: 100%;
+  }
+  
+  .task-form input,
+  .column-form input {
+    flex: 1;
+  }
+  
+  .column {
+    min-width: 280px;
+  }
+} 
+
+.board-title {
+  color: var(--text-primary);
+  font-size: 24px;
+  font-weight: 400;
+  margin-bottom: 24px;
+  cursor: pointer;
+  padding: 4px 8px;
+  border-radius: 4px;
+  transition: background-color 0.2s ease;
+}
+
+.board-title:hover {
+  background-color: rgba(0, 0, 0, 0.05);
+}
+
+.board-title::after {
+  content: '✎';
+  font-size: 18px;
+  margin-left: 8px;
+  opacity: 0;
+  transition: opacity 0.2s ease;
+}
+
+.board-title:hover::after {
+  opacity: 0.5;
+}
+
+.board-title-input {
+  font-size: 24px;
+  font-weight: 400;
+  margin-bottom: 24px;
+  padding: 4px 8px;
+  border: 2px solid var(--primary-color);
+  border-radius: 4px;
+  outline: none;
+  width: auto;
+  min-width: 200px;
+  font-family: inherit;
+  background: white;
+  color: var(--text-primary);
+} 
+
+.column-header-left {
+  display: flex;
+  align-items: center;
+  gap: 8px;
+}
+
+.column-header-right {
+  display: flex;
+  align-items: center;
+  gap: 8px;
+}
+
+.column-emoji {
+  font-size: 20px;
+  cursor: pointer;
+  padding: 4px;
+  border-radius: 4px;
+  transition: background-color 0.2s ease;
 }
 
-.task-form {
-  margin-bottom: 20px;
+.column-emoji:hover {
+  background-color: rgba(0, 0, 0, 0.05);
 }
 
-.task-form input {
+.emoji-picker-container {
+  position: absolute;
+  z-index: 1000;
+  margin-top: 8px;
+}
+
+.color-picker-container {
+  position: absolute;
+  z-index: 1000;
+  background: white;
+  border-radius: 8px;
+  box-shadow: 0 2px 8px rgba(0, 0, 0, 0.15);
   padding: 8px;
-  margin-right: 8px;
-  border: 1px solid #ddd;
-  border-radius: 3px;
+  margin-top: 8px;
+  right: 16px;
 }
 
-.task-form button {
-  padding: 8px 16px;
-  background: #0052cc;
-  color: white;
+.color-options {
+  display: grid;
+  grid-template-columns: repeat(4, 1fr);
+  gap: 8px;
+}
```

Comment:
![kody code-review](https://img.shields.io/badge/kody-code--review-brightgreen)

```suggestion
.color-options {
  display: grid;
  grid-template-columns: repeat(4, 1fr);
  gap: 8px;
}

@media (max-width: 480px) {
  .color-options {
    grid-template-columns: repeat(3, 1fr);
    gap: 4px;
  }
  .color-picker-container {
    right: 8px;
    padding: 4px;
  }
}
```
Consider adding media queries for better mobile responsiveness, particularly for the color-options grid and picker containers which might need adjustments on smaller screens.



<sub>Talk to Kody by mentioning @kody</sub>

<sub>Was this suggestion helpful? React with 👍 or 👎 to help Kody learn from this interaction.</sub><!-- kody-codereview -->&#8203;
&#8203;